src/components/atoms/review-content/review-content.tsx

```diff
@@ -1,10 +1,14 @@
 import Link from 'next/link';
 import './review-content.scss';
 
+/* eslint-disable */
```

Comment:
Do we need to disable the linter here?

Reply:
Adding new lines throws `no-trailing-spaces`.
Not having new lines throws `max-len` (333 > 240)

Looks like @nlisgo is suggesting a different approach above, so have tentatively proposed a different one below.